lightning/src/onion_message/messenger.rs

```diff
@@ -359,9 +365,11 @@ where
 	}
 
 	fn create_blinded_paths<
+		SL: ShortChannelIdLookUp,
 		T: secp256k1::Signing + secp256k1::Verification
 	>(
-		&self, recipient: PublicKey, peers: Vec<PublicKey>, secp_ctx: &Secp256k1<T>,
+		&self, recipient: PublicKey, peers: Vec<PublicKey>, scid_lookup: &SL,
```

Comment:
Since we already have access to the netgraph here, I think it would clarify the purpose of `scid_lookup` to name it something more like `{local,peer}_scid_lookup` and potentially in the trait.